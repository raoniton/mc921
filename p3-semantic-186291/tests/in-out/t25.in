class Main {
    int sim = 0;
    public static void main(String[] args) {
        int n, r, temp;
        print("enter the number = ");
        n = 100100;
        temp = n;
        while(n > 0) {
            r = n % 10;
            this.sum = (this.sum * 10) + r;
            n = n / 10;
        }
        if(temp == this.sum)
            print("palindrome number ");
        else
            print("not palindrome");
    }
}