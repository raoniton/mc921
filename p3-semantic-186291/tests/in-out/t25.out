SemanticError: sum is not defined in the class @ 10:25
